packages/victory-bar/src/helper-methods.ts

```diff
@@ -61,6 +62,14 @@ const getCalculatedValues = (props) => {
   let data = Data.getData(props);
   data = Data.formatDataFromDomain(data, domain, 0);
 
+  if (props.groupComponent.type === VictoryClipContainer) {
+    data = data.map((datum) => {
```

Comment:
Wondering if this should return a new object, of it if doesn't matter since there is already a lot of mutation going on 🤔

Reply:
It definitely won't hurt, I made the update. Thanks.